### PR TITLE
'modification/public map responsiveness mobile

### PR DESCRIPTION
**Pull request by GreenWithMV**
## Description
1) Adds "GeorneyScore" to both public map pages (city, country map) as well as info popup
2) Responsiveness down to mobile sizes for Public city and country maps
  -Includes addition of side menu as in other map pages
3) Z-index issue for new map "sc-controls" buttons is fixed

## Types of changes

- [ X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by hoangviet1993** [comment]
@GreenWithMV I am having trouble checking out the branch because of the `'` in the branch name

File: src/pages/Home/FriendReadonlyMap.js [hunk 1]
```diff
@@ -95,6 +95,16 @@ const FriendReadonlyMap = () => {
     handleTripData(data);
     handleLoaded(true);
   }
+  function geoscoreinfo() {
```
Comment (hoangviet1993) [inline src/pages/Home/FriendReadonlyMap.js]: Should this function be something about setting Swal popup?

File: src/components/Prompts/PopupPrompts.scss [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+@import "./FriendClickedCity/subcomponents/UserTripCard.scss";
+@import "./FilterCityMap.scss";
 
 .popup-container {
     position: fixed;
     display: flex;
     justify-content: center;
     align-items: center;
     left: 0;
+    top: 0;
```
Comment (hoangviet1993) [inline src/components/Prompts/PopupPrompts.scss]: Thanks for this :)

File: src/pages/Home/FriendReadonlyMap.js (hunk 1 above)
Comment (hoangviet1993) [inline src/pages/Home/FriendReadonlyMap.js]: Seems like this function is setting up the swal alert. Should it be named like so? Also for JS, the naming convention should be camelCase right?

File: src/pages/Home/_MapPage.scss [hunk 3]
```diff
@@ -249,6 +246,20 @@
...
+  .sc-controls-left {
+    @media screen and (max-width: 900px) {
+      display: none;
+    }
+  }
+}
+
+#map-readonly {
+  .continent-container {
+    transform: translate(0, 44px);
```
Comment (hoangviet1993) [inline src/pages/Home/_MapPage.scss]: For native CSS styling, please consider using https://autoprefixer.github.io/ (set browserlist config to last 1 version) to add vendor prefixes, also for here and elsewhere :)

File: src/pages/Home/_MapPage.scss [hunk 4]
```diff
@@ -274,7 +285,7 @@
     }
   }
   .country-svg {
-    fill: #6E7377;
+    fill: #6e7377;
```
Comment (hoangviet1993) [inline src/pages/Home/_MapPage.scss]: If this color (and other color) is used globally, I think we should consider setting a global color css class?

File: src/pages/Home/subcomponents/_CityMap.scss [hunk 5]
```diff
@@ -691,4 +705,11 @@
...
   .mapboxgl-ctrl-geocoder--icon-close {
     margin-top: 4px;
+  }
+
+  .city-map-readonly {
+    .city-new-side-menu {
+      top: 0;
+      height: 100%;
+    }
   }
```
Comment (hoangviet1993) [inline src/pages/Home/subcomponents/_CityMap.scss]: New line for this file please 👍

File: src/pages/Home/subcomponents/_CityMap.scss [hunk 6]
```diff
@@ -272,18 +272,32 @@
...
   height: 50px;
-  z-index: 200;
+  width: 400px;
+  z-index: 2;
   border-bottom: 2px solid #a7e1ff;
+  display: flex;
+  justify-content: center;
+  @media screen and (max-width: 450px) {
+    font-size: 24px;
+    transform: translateX(20px);
```
Comment (hoangviet1993) [inline src/pages/Home/subcomponents/_CityMap.scss]: Add vendor prefix for translateX

File: src/pages/Home/_MapPage.scss [hunk 7]
```diff
@@ -249,6 +246,20 @@
...
+    }
+  }
+}
+
+#map-readonly {
+  .continent-container {
+    transform: translate(0, 44px);
+  }
+  .rsm-svg {
+    transform: translateY(40px);
```
Comment (hoangviet1993) [inline src/pages/Home/_MapPage.scss]: Add vendor prefix for translateY.

File: src/pages/Home/subcomponents/_NewUserCity.scss [hunk 8]
```diff
@@ -312,14 +313,20 @@
         }
         @media screen and (max-width: 500px) {
           min-width: 350px;
+          transform: translateX(20px);
```
Comment (hoangviet1993) [inline src/pages/Home/subcomponents/_NewUserCity.scss]: Add vendor prefix for translateX.

File: src/pages/Home/FriendReadonlyMap.js (hunk 1 above)
Comment (GreenWithMV) [reply src/pages/Home/FriendReadonlyMap.js]: yep, will change to geoScoreSwal

File: src/pages/Home/_MapPage.scss (hunk 4 above)
Comment (GreenWithMV) [reply src/pages/Home/_MapPage.scss]: Yes, might be worth a separate issue where we do that for all of the colors

File: src/pages/Home/subcomponents/_CityMap.scss (hunk 5 above)
Comment (GreenWithMV) [reply src/pages/Home/subcomponents/_CityMap.scss]: Sorry, what do you mean?

Comment (hoangviet1993) [reply src/pages/Home/subcomponents/_CityMap.scss]: Ah I was talking about inserting a new line at the end of this file.

**Pull request merged**